openrouteservice/src/main/java/org/heigit/ors/api/requests/routing/RouteRequestHandler.java

```diff
@@ -504,4 +512,12 @@ private  boolean convertSetFlexibleMode(boolean useContractionHierarchies) throw
 
         return avoidCountryIds;
     }
+
+    private double convertUserSpeed(Double userSpeed) throws ParameterValueException{
```

Comment:
I used the naming from the previous functions for a consistency. But since my feature is only a double I did not nedd to convert it to sth. Do you have any suggestions for a name?

Reply:
What about validateUserSpeed?